Cargo.toml

```diff
@@ -21,9 +22,6 @@ members = [
     "demos/macroquad/server",
     "demos/macroquad/shared",
     "demos/macroquad/client",
-    "demos/hecs/server",
```

Comment:
I commented these out because of a `syn` conflict, I think that might be resolved now. Should add these back to the workspace.